client/gutenberg/extensions/classic-block/edit.jsx

```diff
@@ -23,22 +23,56 @@ export class ClassicEdit extends Component {
 	}
 
 	storeEditor = ref => {
+		//By the time editor lifecycle events like PostProcess fire, this editor instance is already marked for removal
+		//so any content manipulation doesn't get reflected in this block.
+		//
+		//For now, when we detect an unmount, get content from the editor marked to be destroyed and update block attributes.
+		//https://github.com/Automattic/wp-calypso/issues/29643
+		if ( ! ref && this.editor ) {
+			const content = this.editor._editor.getContent();
+			this.props.setAttributes( { content } );
+		}
 		this.editor = ref;
 	};
 
-	debouncedOnContentChange = debounce( () => {
-		const rawContent = this.editor.getContent( { format: 'raw' } );
+	componentDidUpdate( prevProps ) {
+		const {
+			attributes: { content },
+		} = this.props;
+
+		if ( this.editor && prevProps.attributes.content !== content ) {
+			this.editor.setEditorContent( content || '' );
+		}
+	}
+
+	updateBlockContentAndBookmark = () => {
+		if ( ! this.editor ) {
+			return;
+		}
+		const editor = this.editor._editor;
+
+		this.bookmark = editor.selection.getBookmark( 2, true );
```

Comment:
can we explain the parameters here since it'd be hard to follow `getBookmark` to see its source?

Reply:
Sure, it's a copy pasta from the core classic block, of which I had to dig into source to see the actual value. It returns an offsetBookmark. I'll add a comment.
 
https://www.tiny.cloud/docs/api/tinymce.dom/tinymce.dom.bookmarkmanager/#getbookmarkhttps://github.com/tinymce/tinymce/blob/146c3c7c98206f21fbe12f61537ff42dbe5c6c3a/src/core/main/ts/bookmark/GetBookmark.ts#L227
https://github.com/tinymce/tinymce/blob/146c3c7c98206f21fbe12f61537ff42dbe5c6c3a/src/core/main/ts/bookmark/GetBookmark.ts#L154

Reply:
Updated in 052ffe1